Presentation.Web/Content/less/kitos.less

```diff
@@ -1661,4 +1661,8 @@ tbody.bordered > tr > td {
 
 .wide-modal .modal-dialog {
     width: 1000px !important;
+}
+
+.color-maroon{
```

Comment:
perhaps introduce a term in sead... perhaps: input-validation-error or something like that